examples/crm/src/Header.tsx

```diff
@@ -45,10 +46,18 @@ const Header = () => {
             <AppBar position="static" color="primary">
                 <Toolbar variant="dense">
                     <Box flex={1} display="flex" justifyContent="space-between">
-                        <Box display="flex" alignItems="center">
+                        <Box
+                            display="flex"
+                            alignItems="center"
+                            onClick={() => redirect('/')}
```

Comment:
If you use an `onClick` handler, the target doesn't appear when hovering the link. Onstead, use react-router's `Link` component. It will also change the mouse pointer automatically. 

Reserve navigation with click handlers for conditional navigation.